### DISCUSSION TITLE
Control-Lines-BackgroundDamper line style (setup cannot be kept)

### DISCUSSION THREAD

**Discussion by mansonchan888** [artisan-roaster-scope/artisan#1131] [Q&A]
Cannot set up the line style 

I try to set my the line style or colour different between the Damper and its backgrounddamper, it shows the changes after I press "apply" but it turn back to the pre-set color after I press "ok"
It happen to backgroudBurner and backgrounddrum. 

<img width="1016" alt="WeChatac6c625b80c72fa98ecb1a7a73b90ffd" src="https://user-images.githubusercontent.com/119805088/226084904-d6d90f21-5a00-491a-9ab2-a2c4b6318bc0.png">

**Comment by MAKOMO** [comment]
Artisan does not support different designs for background events. The colors of background elements are always derived from their foreground counterparts  using the `Opaquness` setting to avoid confusions. This will not change.

The dialog you show is generic and only changes the attributes of the current drawn artists of the underlying graphing tool.

**Comment by mansonchan888** [comment]
thankyou for the information
